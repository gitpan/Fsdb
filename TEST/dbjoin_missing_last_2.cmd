prog='dbjoin'
args='-S -e 0 -a --input TEST/dbjoin_missing_last_2.in-2 --input - low'
in=TEST/dbjoin_missing_last_1.in
cmp='diff -cb '
