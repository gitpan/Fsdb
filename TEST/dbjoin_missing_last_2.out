#fsdb low noshare histogram
1	7	47
2	39	82
3	86	97
4	137	85
5	150	59
6	101	36
7	38	10
8	13	7
9	6	0
#   | dbjoin -S -e 0 -a --input TEST/dbjoin_missing_last_2.in-2 --input - low
